config/common.php

```diff
@@ -10,23 +10,11 @@
 use Yiisoft\Log\Logger;
 
 return [
-    LoggerInterface::class => function (ContainerInterface $container) {
-        $logger = $container->get(Logger::class);
+    LoggerInterface::class => Logger::class,
```

Comment:
Do we need the logger here at all? Isn't it used in tests only?

Reply:
AssetsManager used Logger.

Reply:
https://github.com/yiisoft/assets/blob/72126623b402d9245924c5ee397cd75f3655548c/src/AssetManager.php#L64